### PR TITLE
[Dx 720] - Update Release Landing Page With Latest Component Versions

### PR DESCRIPTION
**Pull request by dcs3spp**
[DX-720](https://tyktech.atlassian.net/browse/DX-720)

[preview - release landing page](https://deploy-preview-3640--tyk-docs.netlify.app/docs/nightly/developer-support/tyk-release-summary/overview)

Update Release landing page to include latest version info, with links to GHub and DockerHub images

[DX-720]: https://tyktech.atlassian.net/browse/DX-720?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dcs3spp** [comment]
Hi @letzya I have updated the PR with the changes we discussed earlier today

@ilijabojanovic would you be able to review the dependencies page?

File: tyk-docs/content/developer-support/third-party-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Third Party Dependencies
```
Comment (letzya) [inline tyk-docs/content/developer-support/third-party-dependencies.md]: @dcs3spp should we mention that this page is not just a list of dependencies but also the supported versions?

File: tyk-docs/content/developer-support/third-party-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+description: This page summarises 3rd Party Dependencies
+layout: dependencies
+---
+
+This page summarises the third party dependencies used by Tyk Open Source and licensed components.
```
Comment (letzya) [inline tyk-docs/content/developer-support/third-party-dependencies.md]: Same as above, @dcs3spp, we  should add that wording explaining that this page provides the supported versions?

Comment (letzya) [inline tyk-docs/content/developer-support/third-party-dependencies.md]:
```suggestion
This page details the third-party dependencies that Tyk components rely on, including the relevant supported and tested versions
```

File: tyk-docs/data/dependencies.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
+{
+    "data": [
+        {
```
Comment (letzya) [inline tyk-docs/data/dependencies.json]: Python

**Comment by dcs3spp** [comment]
@letzya I have resolved the comments relating to 3rd party dependencies since they will be be added to release notes. I have updated the latest components page with banner and added some stylesheet changes. If all is ok, can this now be merged and merged?

File: tyk-docs/assets/scss/releases.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.release-card:hover {
+  box-shadow: 0 0 15px rgba(0, 0, 0, 0.3);
+  transform: scale(1.05);
+}
+
+.release-description {
+  font-size: 0.75rem;
+  font-family: 'Roboto', sans-serif;
```
Comment (yurisasuke) [inline tyk-docs/assets/scss/releases.scss]: We decided to change font in the whole site to `inter ` (rebranding stuff)

File: tyk-docs/assets/scss/releases.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+.release-link {
+  font-size: 0.75rem;
+  display: block;
+  text-decoration: none !important;
+}
+
+.release-text {
+  font-size: 0.75rem;
+  font-family: 'Roboto', sans-serif;
```
Comment (yurisasuke) [inline tyk-docs/assets/scss/releases.scss]: here too

File: tyk-docs/assets/scss/releases.scss [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  font-weight: 400;
+  line-height: 20px; 
+}
+
+.release-title {
+  font-size: 1rem;
+  font-family: 'Roboto', sans-serif;
```
Comment (yurisasuke) [inline tyk-docs/assets/scss/releases.scss]: here too

**Review by yurisasuke — APPROVED** [review]

File: tyk-docs/themes/tykio/layouts/partials/developer-support/latest-releases/release-summary.html [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+                                    {{- end -}}
+                                {{- end -}}
+                                {{- $title -}}
+                            </div>
+                            <!-- body -->
+                            <div class="card-body">
+                                <div class="row">
+                                    <div class="col-md-12 mt-0 mb-0">
+                                        <p class="release-text">
+                                            Discover more in the <a href="{{ $component.data.home | relURL }}">docs</a>
```
Comment (letzya) [inline tyk-docs/themes/tykio/layouts/partials/developer-support/latest-releases/release-summary.html]:
```suggestion
                                           Click to <a href="{{ $component.data.home | relURL }}"> discover more</a>
```

File: tyk-docs/themes/tykio/layouts/partials/developer-support/latest-releases/release-summary.html [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+                    
+                            <!-- Footer -->
+                            <div class="card-footer bg-transparent border-0 p-0 d-flex">
+                                <a
+                                    href="{{ $releaseNotesPath | relURL }}"
+                                    target="_blank"
+                                    rel="noopener noreferrer"
+                                    class="home_btn sec-btn btn-block w-100 text-center"
+                                >
+                                    Release Notes
```
Comment (letzya) [inline tyk-docs/themes/tykio/layouts/partials/developer-support/latest-releases/release-summary.html:118]: The text "Release Notes" stands out while the product names such as "MDCB v2.4.1" are much less.
Maybe we can make the font smaller? or other ideas?

**Comment by letzya** [comment]
@rahmatlatif @vverbani could you please review this?

File: tyk-docs/themes/tykio/layouts/developer-support/release-summary.html [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+{{ end }}
+
+
+<section class="home-section">
+  <div class="container-xl p-0">
+    <div class="align-items-center component-heading">
+      <div>
+        <div class="home-section-heading">Discover our releases</div>
+        <div class="fs-16 font-400 fs-normal text-sec lh-normal">
+          Find out more about the latest releases for all our platforms below
```
Comment (letzya) [inline tyk-docs/themes/tykio/layouts/developer-support/release-summary.html]:
```suggestion
          Explore the most recent releases across all our platforms listed below
```

Comment (letzya) [reply tyk-docs/themes/tykio/layouts/developer-support/release-summary.html]: This title is smaller than the next "Tyk components" which is h2. Was that intended?

**Review by letzya — COMMENTED** [review]
Except for a couple of comments, looks good!!! we need to get someone from the support or commercial team to tell us if that's ok and also @allaneyles

**Comment by letzya** [comment]
@dcs3spp can I please suggest changing the following:
dashboard to tyk dashboard, and so on, in all the components
portal - to tyk enterprise developer portal

order in oss: 
GW,  operator, pump, sync, tib,  helm.

File: tyk-docs/themes/tykio/layouts/developer-support/release-summary.html (hunk 9 above)
Comment (letzya) [reply tyk-docs/themes/tykio/layouts/developer-support/release-summary.html]: Actually, I think this heading and one sentence is not needed at all. We have the intro

File: tyk-docs/content/developer-support/tyk-release-summary/overview.md [hunk 10]
```diff
@@ -1,52 +1,11 @@
...
-{{< badge href="https://github.com/TykTechnologies/tyk-operator/releases" image="/img/logos/tyk-logo-opensource.svg" imageStyle="object-fit:contain" alt="Tyk Operator Release Notes">}}
-**Tyk Operator**
-{{< /badge >}}
-
-{{< badge href="https://github.com/TykTechnologies/tyk-sync/releases" image="/img/logos/tyk-logo-opensource.svg" imageStyle="object-fit:contain" alt="Tyk Sync Release Notes">}}
-**Tyk Sync**
-{{< /badge >}}
-
-{{< /grid >}}
+Explore the latest tagged open-source releases on our GitHub repositories. Enhance integration with Docker images for seamless deployment. Detailed insights, including features, enhancements and bug fixes, are in our comprehensive release notes—a vital resource for developers evaluating upgrades.
```
Comment (letzya) [inline tyk-docs/content/developer-support/tyk-release-summary/overview.md]: why open source and GH if this page is for all the components?
I'd also drop "tagged" as it's too technical.

Not sure i understand the intro.

Comment (dcs3spp) [reply tyk-docs/content/developer-support/tyk-release-summary/overview.md]: @letzya 

```suggestion
Download the latest releases for our Tyk components. Links are provided to download docker images and source code ( where applicable for our open-source components). Detailed insights, including features, enhancements and bug fixes, can be found in our comprehensive release notes. These are a vital resource for developers evaluating upgrades.
```

File: tyk-docs/themes/tykio/layouts/developer-support/release-summary.html (hunk 9 above)
Comment (dcs3spp) [reply tyk-docs/themes/tykio/layouts/developer-support/release-summary.html]: Done and made suggestion for refactored intro paragraph

File: tyk-docs/themes/tykio/layouts/partials/developer-support/latest-releases/release-summary.html (hunk 8 above)
Comment (dcs3spp) [reply tyk-docs/themes/tykio/layouts/partials/developer-support/latest-releases/release-summary.html:118]: Have reduced the font size text of the button

**Pull request merged**